yamale/schema/validationresults.py

```diff
@@ -26,8 +26,13 @@ def __init__(self, data, schema, errors):
     def __str__(self):
         if self.isValid():
             error_str = "'%s' is Valid" % self.data
-        else: 
-            head_line = "Error validating data '%s' with '%s'\n\t" % (self.data, self.schema)
+        else:
+            head_line_bits = ["Error validating data"]
+            if self.data:
+                head_line_bits.append("'{}'".format(self.data))
+            if self.schema:
+                head_line_bits.append("with schema '{}'".format(self.schema))
+            head_line = " ".join(head_line_bits)
             error_str = head_line + '\n\t'.join(self.errors)
```

Comment:
Thanks for the message formatting. Looks like the `\n\t` after the `head_line` was missed in refactoring.

Reply:
Fixed in https://github.com/23andMe/Yamale/pull/105/commits/23b49e6d9f807584441eab7ca8b26c867cc1a405#diff-cbb0754bd27ec59a69abd0f651e48c3b